plugins/ipam/host-local/backend/disk/backend.go

```diff
@@ -65,7 +65,7 @@ func (s *Store) Reserve(id string, ip net.IP, rangeID string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	if _, err := f.WriteString(strings.TrimSpace(id)); err != nil {
+	if _, err := f.WriteString(strings.TrimSpace(id) + "\n" + ifname); err != nil {
```

Comment:
@mccv1r0 your concerns WRT Windows are probably valid.  We could use \r\n if you want to be more compatible there.

Reply:
Nobody seemed to like my suggestion of using ASCII BEL :-)